models/articles.model.js

```diff
@@ -3,12 +3,21 @@ const db = require('../db/connection');
 
 exports.selectArticleById = (article_id) => {
     
-    let query = ` 
-    SELECT * FROM articles 
-    WHERE article_id = $1
-    ;`;
+    let newQuery = `
+    SELECT articles.article_id, articles.title, articles.topic, articles.author, articles.body, 
+    articles.created_at, articles.votes, COUNT(comments.article_id) AS comment_count
+    
+    FROM articles
+  
+    JOIN comments 
+    ON articles.article_id = comments.article_id
+
+    WHERE articles.article_id = $1
+
+    GROUP BY articles.article_id, articles.title, articles.topic, articles.author, articles.body,
```

Comment:
This looks like a good way to do it to me. You could try grouping by just the articles_id (I think it would work) to save some code.

Reply:
tried it got few errors, then I kept it the same for now, just to get going on the other tasks